pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/UnsafeUtils.java

```diff
@@ -0,0 +1,262 @@
+   */
+  static sun.misc.Unsafe getUnsafe() {
+    sun.misc.Unsafe unsafe = null;
+    try {
+      unsafe =
+          AccessController.doPrivileged(
+              new PrivilegedExceptionAction<sun.misc.Unsafe>() {
+                @Override
+                public sun.misc.Unsafe run() throws Exception {
+                  Class<sun.misc.Unsafe> k = sun.misc.Unsafe.class;
+                  for (Field f : k.getDeclaredFields()) {
+                    f.setAccessible(true);
+                    Object x = f.get(null);
+                    if (k.isInstance(x)) {
+                      return k.cast(x);
+                    }
+                  }
+                  // The sun.misc.Unsafe field does not exist.
+                  return null;
+                }
+              });
+    } catch (Throwable e) {
+      // Catching Throwable here due to the fact that Google AppEngine raises NoClassDefFoundError
+      // for Unsafe.
+    }
+    return unsafe;
+  }
+  /** Get a {@link MemoryAccessor} appropriate for the platform, or null if not supported. */
+  private static MemoryAccessor getMemoryAccessor() {
+    if (UNSAFE == null) {
+      return null;
+    }
+    return new JvmMemoryAccessor(UNSAFE);
+  }
+  private static boolean supportsUnsafeArrayOperations() {
+    if (MEMORY_ACCESSOR == null) {
+      return false;
+    }
+    return MEMORY_ACCESSOR.supportsUnsafeArrayOperations();
+  }
+  private static boolean supportsUnsafeByteBufferOperations() {
+    if (MEMORY_ACCESSOR == null) {
+      return false;
+    }
+    return MEMORY_ACCESSOR.supportsUnsafeByteBufferOperations();
+  }
+  /** Finds the address field within a direct {@link Buffer}. */
+  private static Field bufferAddressField() {
+    Field field = field(Buffer.class, "address");
+    return field != null && field.getType() == long.class ? field : null;
+  }
+
+  /**
+   * Returns the offset of the provided field, or {@code -1} if {@code sun.misc.Unsafe} is not
+   * available.
+   */
+  private static long fieldOffset(Field field) {
+    return field == null || MEMORY_ACCESSOR == null ? -1 : MEMORY_ACCESSOR.objectFieldOffset(field);
+  }
+  /**
+   * Gets the field with the given name within the class, or {@code null} if not found.
+   */
+  private static Field field(Class<?> clazz, String fieldName) {
+    Field field;
+    try {
+      field = clazz.getDeclaredField(fieldName);
+    } catch (Throwable t) {
+      // Failed to access the fields.
+      field = null;
+    }
+    return field;
+  }
+  private abstract static class MemoryAccessor {
+    sun.misc.Unsafe unsafe;
+    MemoryAccessor(sun.misc.Unsafe unsafe) {
+      this.unsafe = unsafe;
+    }
+    public final long objectFieldOffset(Field field) {
+      return unsafe.objectFieldOffset(field);
+    }
+    public final int arrayBaseOffset(Class<?> clazz) {
+      return unsafe.arrayBaseOffset(clazz);
+    }
+
+    /** Relative Address Operations ---------------------------------------------
```

Comment:
What happened to the comment here, normally we have section separators as single line comment more like:
```
\\ ---- Relative Address Operations ---------------------------------------------
```
Also Java doc should either be one line or not so either
```
/** ddkjj dkjkdj  djkj */
```
or multiline with nothing on first line after `/**' like
```
/** 
 * ddkjj dkjkdj  djkj 
 */
```

Reply:
Fixed.